### ISSUE TITLE
Cannot upgrade to any later versions of Cura after installing 5.0

### ISSUE THREAD

**Issue by myq1960** [Ultimaker/Cura#18178]
### Cura Version

5.0

### Operating System

Windows 10

### Printer

Ender 3 V2

### Reproduction steps

Ever since I installed Cura 5.0.0, I have not been able to update to a newer version (5.1 and up). I download these versions and they seem to install ok but when I try to run them nothing happens. I've tried completely removing my existing version of Cura 5.0 (including all Cura folders) and start from scratch with installing the newer version but that still doesn't help. 

### Actual results

Still am not able to run an versions of Cura after 5.0

### Expected results

I should be able to run later versions of Cura

### Add your .zip and screenshots here ⬇️

I don't have any screenshots for files to provide.

**Comment by Vandresc** [comment]
Hi @myq1960, thank you for your feedback! Did you try running Cura from the installation directory, by clicking UltimakerCura.exe?

**Comment by myq1960** [comment]
Yes I did.  I even right clicked and selected “run as administrator” but still nothing happens.

**Comment by HellAholic** [comment]
Hey @myq1960 , we have a set of usual suspects when it comes to cura not starting. But since it's a device specific issue, we cannot for sure say what is blocking your upgrade. Please try the following steps and let us know if they worked for you.

1. Do you have multiple monitors? Does Cura start if you disable one of the monitors?
2. Do you have two monitors placed above each other? Can you try this workaround from the community? [Cura 5.2.2 stopped working](https://community.ultimaker.com/topic/42633-cura-522-stopped-working/#comment-318105) 
3. Do you have GPU Tweak OSD overlay enabled? Does Cura start if you disable that software?
4. Do you have a AMD Graphics card? Can you try this workaround? [Support Community](https://support.makerbot.com/s/article/1667412437627) 
5. Do you have the Nahimic audio enhancement service enabled? Can you try this workaround? [Cura 5 Beta doesn't support main display on second monitor · Issue #12102 · Ultimaker/Cura](https://github.com/Ultimaker/Cura/issues/12102#issuecomment-1157907665) 
6. If your problem is related to scaling, can you try this workaround from the community? [Ultimaker Cura 5.1.0 Window Scaling Problem](https://community.ultimaker.com/topic/41458-ultimaker-cura-510-window-scaling-problem/#comment-311838) 
7. Try removing the configuration folders for Cura and launching Cura again. You can find these folders in "OperatingSystemDrive (C):\Users\Your username\AppData\Roaming\cura\" and "Your Operating System Drive (C):\Users\Your username\AppData\Local\cura\".
